dtests/src/test/java/io/snappydata/hydra/clusterRecovery/newNodeStartingFirstTest.conf

```diff
@@ -0,0 +1,58 @@
+hydra.Prms-testRequirement = "Test to verify cluster recovery when a new server/lead node starts first in the existing cluster.";
+hydra.Prms-testDescription = "Test to verify cluster recovery when a new server/lead node starts first in the existing cluster. ";
```

Comment:
Check the detailed test description in all confs and add wherever it is missing.